### ISSUE TITLE
[Deno] mergeRight not found on import

### ISSUE THREAD

**Issue by iwinux** [selfrefactor/rambda#633]
## How to reproduce

```sh
deno cache 'https://x.nest.land/rambda@7.1.0/mod.ts'
```

## Expected

It downloads the package.

## Actual

```
error: Import 'https://arweave.net/TuHF0i98hvaUr5zqtqBTc3cgLqSvqf7aOCN8qSB3GVw/src/mergeRight' failed, not found.
    at https://arweave.net/TuHF0i98hvaUr5zqtqBTc3cgLqSvqf7aOCN8qSB3GVw/src/merge.js:1:35
```

## Workaround

Switch to 7.0.6 instead:

```sh
deno cache https://x.nest.land/rambda@7.0.6/mod.ts
```

**Comment by selfrefactor** [comment]
I will check it out. Thanks for bringing it up.

**Comment by selfrefactor** [comment]
Fixed with `7.1.4`
